notebooks/metrics_demo.ipynb

```diff
@@ -88,7 +93,10 @@
     "## Trustworthiness\n",
```

Comment:
Maybe "Clustering and Dimensionality Reduction and Related Metrics" to emphasize the algo side too?